server/graphql/general/src/queries/barcode.rs

```diff
@@ -0,0 +1,108 @@
+use async_graphql::*;
+use graphql_core::{
+    generic_filters::{EqualFilterNumberInput, EqualFilterStringInput},
+    pagination::PaginationInput,
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use graphql_types::types::BarcodeConnector;
+use repository::{BarcodeFilter, BarcodeSort, BarcodeSortField};
+use repository::{EqualFilter, PaginationOption};
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    barcode::get_barcodes,
+};
+
+#[derive(Enum, Copy, Clone, PartialEq, Eq)]
+#[graphql(remote = "repository::BarcodeSortField")]
+pub enum BarcodeSortFieldInput {
+    Id,
+    Barcode,
```

Comment:
I've noticed we've used `value` for barcode field and filter, but for sort it's Barcode, I kind of like barcode instead of value tbh, either way, if it's value or barcode would be great to have it consistent

Reply:
I was resisting using `barcode.barcode` as per mSupply implementation. `value` isn't great, but it is less repetitive.
Might review that though in #1629